#215 Created Return Button that Appears at the end of each lab

dm9965 opened the pull request:
- [ ] No discrepancies across browsers. Eg. chrome vs safari
- [ ] Pages pass W3C Validation for HTML (https://validator.w3.org)
- [ ] CSS (https://jigsaw.w3.org/css-validator/)
- [ ] Google sign-in is operational
- [ ] Accessibility functions
- [ ] Pages can scale without distorting page
- [ ] No dead links
- [ ] Navbar is consistent across the site
- [ ] Pages are screen-reader accessible
- [ ] Contrast meets standards for accessibility
- [ ] Pages are keyboard accessible
- [ ] Passes WAVE Evaluation Tool chrome extension

dm9965 committed:
#215 Created Return Button that Appears at the end of each lab

Mster115 reviewed (changes requested):
BLOCKED: Until Lab 7 merge to account for hidden certificate check, this would entail if hiddenCertificate then this button is not shown

Mster115 commented:
@dm9965 lab 7 merged

heathermoses commented on client/src/assets/stylesheets/pages/quiz.scss:
```diff
@@ -184,6 +184,12 @@
+    margin: 40px;
+    padding:15px;
+    background-color: #3f51b5;
+  }
```
Would it be possible to make the button fit to the text like the next and previous buttons? Right now the width seems pretty big.

![Screenshot 2023-04-03 at 9 47 03 AM](https://user-images.githubusercontent.com/54957497/229529409-7d1e93e4-78c3-48ba-ba43-8cb8eabd7e92.png)

dm9965 replied:
I'll fix it this weekend!

dm9965 replied:
It was fixed this weekend! 🤪

dm9965 committed:
VimCrypt~03!X1¶,Í$ù!íÒâi[+ddøþ6<²h:$°ÃåÕv··yh';9ÊXÏn~õÿf¾'"¾Üß7¡#Ï+¸ÛPEÉØÉX°Ҥ]ޠ¿#¼{ÀMӬ>ßÃ	È+ÃçÆ<	Xéñ[0ó¹÷¢¦-CZ®Aiî±]½çEJÎìdAüR3¤ùᑢ®.¼é®ðiï«ÿö9±OG9/nR>Åzu«zýÊ?F©µ®ÆôQ5+é_<Þ_YZ
2>wb/}VTÊ}"HZï©Rö«¤4¥2O¶×<åá½óTr©/ið± þ^ÐÉY9îÅ	|á1df`LæK{ Vtc7M$eWOþõéĶÜÐôûÜYdÎc<[ç9A²µi
¦

dm9965 committed:
Updated Button to Work in footer

Commit:
Updated quizCompleted hook to App.js so have top down useability for new button

Commit:
Committing fixed return home button for end of labs

heathermoses commented on client/src/components/quiz/App.js:
```diff
@@ -95,7 +94,8 @@ const QuizHandler = (props) => {
     scoreResults();
-    setQuizCompleted(true);
+    props.setQuizCompleted(true);
+    console.warn(props.quizCompleted);
```
Nit: I think this console warn can be removed

heathermoses reviewed (approved):
Tested on all labs, works for me!

heathermoses committed:
remove console log
